instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/TracingExecutionInterceptor.java

```diff
@@ -35,101 +42,121 @@
   static final ExecutionAttribute<Span> SPAN_ATTRIBUTE =
       new ExecutionAttribute<>("io.opentelemetry.auto.Span");
 
+  static final ExecutionAttribute<RequestType> REQUEST_TYPE_ATTRIBUTE =
+      new ExecutionAttribute<>("io.opentelemetry.auto.aws.RequestType");
+
   static final String COMPONENT_NAME = "java-aws-sdk";
 
-  private final Kind kind;
+  private final SpanFactory spanFactory;
+
+  private final Map<RequestType, SdkRequestDecorator> decoratorsMap = mapDecorators();
+  private final Map<String, String> fieldToAttribute = mapFieldToAttribute();
```

Comment:
Let's make the constant-type things static

Reply:
Done (with relevant name change).